roles/intern.md

```diff
@@ -0,0 +1,60 @@
+---
+type: Resource
+---
+
+# Internship program
+
+Our program offers you the ability to work on high value public code projects used around the world, and improving the role of technology in society. You will be helping to make public-purpose software and policy – used for example in cities and other public organizations – be more open and collaborative. 
+
+[The Foundation for Public Code](https://publiccode.net) helps public organizations develop and maintain software and policy together. We provide review of codebases that are in our stewardship to help developers build trusted and reliable software that can be widely reused and foster a strong community around it.
+
+We test against the [Standard for Public Code](http://standard.publiccode.net/) to make codebases understandable, reusable and community based. Our standards are high, as well as opinionated, to ensure codebases can be trusted as the infrastructure of society and improve continually, and policy makers can keep technology and architectural choices open.
+
+## What interns do
+
+As a prospective intern we ask you to send us a proposal on how you see yourself fit best within the Foundation's activities, based on the information below, your skills and your interests. 
+
+### Join as junior codebase stewards
```

Comment:
I'd actually suggest using 'intern' in stead of 'junior' here because we might want to hire actual juniors on larger codebases.

Reply:
done